Summary/analyze/util.py

```diff
@@ -28,7 +28,7 @@
 MYSQL_CONFIG = {
     "host": "localhost",
     "user": "djangouser",
-    "password": "paSSw0#d",
+    "password": "password",
```

Comment:
I'm assuming the user can locally change the password anyways right? Why are you changing it in all PRs? Also, is it possible to have a single MYSQL config file with this info?

Reply:
If anyone clones the repo and tries to run it following the ReadMe then it fails, because the password provided in the ReadMe is different than what's inside the files. This is the only file that has this info.

Reply:
you can refactor this in a separate PR